build.sbt

```diff
@@ -172,11 +264,23 @@ val jsdocs = project
   )
   .enablePlugins(ScalaJSPlugin, ScalaJSBundlerPlugin)
 
+lazy val worksheets = project
```

Comment:
I needed to extract the worksheet suites in order not to depend on scala js projects.